### PR TITLE
react-react1-week1

### PR DESCRIPTION
**Pull request by Hanouj**

### CODE REVIEW

File: react/react1/week1/react1week1/src/TodoList.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        description:"Eat breakfast" ,
+        deadLineDate: "Fri Sep 15 2017",
+       
+      },
+  ];
+export function TodoList() {
+    return(
+    <>
+   
+        {todoList.map((item) => <TodoListItems descp={item.description} date={item.deadLineDate} key={item.id}/> )}
```
Comment (annamourasobral) [inline react/react1/week1/react1week1/src/TodoList.jsx:24]: those props aren't relevant here, use them inside the component

```suggestion
        {todoList.map((item) => <TodoListItems item={item} key={item.id}/> )}
```

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hey, @Hanouj 
Your app is working but needs some improvement.
I really missed the styling here, we are front-end developers! 
Happy coding! 👩🏻‍💻

**Review by annamourasobral — APPROVED** [review]
Thank you for the updates =)

File: react/react1/week1/react1week1/src/Page.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import Header from "./Header"
```
Comment (Hailegebriel22) [inline react/react1/week1/react1week1/src/Page.jsx:1]: again it would be nice to start with `import React from 'react';`

**Review by Hailegebriel22 — COMMENTED** [review]
@Hanouj  Good Job. You already have received feedback from @annamourasobral, I have added a comment I hope it will be helpful.

**Pull request merged**